stock_reserve/view/product.xml

```diff
@@ -1,33 +1,31 @@
 <?xml version="1.0" encoding="utf-8"?>
-<openerp>
-    <data noupdate="0">
+<odoo>
     
-        <record model="ir.ui.view" id="product_template_form_view_reservation_button">
-            <field name="name">product.template.reservation.button</field>
-            <field name="model">product.template</field>
-            <field name="inherit_id" ref="product.product_template_only_form_view"/>
-                <field name="arch" type="xml">
-                <xpath expr="//div[@name='buttons']" position="inside">
-                    <button class="oe_inline oe_stat_button" name="action_view_reservations"
-                        type="object" groups="base.group_sale_salesman" icon="fa-lock">
-                        <field string="Stock Reservations" name="reservation_count" widget="statinfo" />
-                    </button>
-                </xpath>
-            </field>
-        </record>
+<record model="ir.ui.view" id="product_template_form_view_reservation_button">
+    <field name="name">product.template.reservation.button</field>
+    <field name="model">product.template</field>
+    <field name="inherit_id" ref="product.product_template_only_form_view"/>
+        <field name="arch" type="xml">
+        <xpath expr="//div[@name='button_box']" position="inside">
+            <button class="oe_inline oe_stat_button" name="action_view_reservations"
+                type="object" attrs="{'invisible':[('type', '!=', 'product')]}" icon="fa-lock">
+                <field string="Stock Reservations" name="reservation_count" widget="statinfo" />
+            </button>
+        </xpath>
+    </field>
+</record>
 
-        <record model="ir.ui.view" id="product_product_form_view_reservation_button">
-            <field name="name">product.template.reservation.button</field>
-            <field name="model">product.product</field>
-            <field name="inherit_id" ref="product.product_template_form_view"/>
-            <field name="arch" type="xml">
-                <xpath expr="//div[@name='buttons']" position="inside">
-                    <button class="oe_inline oe_stat_button" name="action_view_reservations"
-                        type="object" groups="base.group_sale_salesman" icon="fa-lock">
-                        <field string="Stock Reservations" name="reservation_count" widget="statinfo" />
-                    </button>
-                </xpath>
-            </field>
-        </record>
-    </data>
-</openerp>
+<record model="ir.ui.view" id="product_product_form_view_reservation_button">
+    <field name="name">product.template.reservation.button</field>
+    <field name="model">product.product</field>
+    <field name="inherit_id" ref="product.product_template_form_view"/>
+    <field name="arch" type="xml">
+        <xpath expr="//div[@name='button_box']" position="inside">
+            <button class="oe_inline oe_stat_button" name="action_view_reservations"
```

Comment:
This button is not being showed for the product.product.
![image](https://cloud.githubusercontent.com/assets/23449160/24157279/43cb0604-0e59-11e7-90ed-7567c30aa824.png)
I accessed it directly from the reservation.